### ISSUE TITLE
Reconsider 'User agents should not display non-persistent notification in a platform’s "notification center"' part?

### ISSUE THREAD

**Issue by saschanaz** [whatwg/notifications#203]
### What is the issue with the Notifications API Standard?

>User agents should not display [non-persistent notification](https://notifications.spec.whatwg.org/#non-persistent-notification) in a platform’s "notification center" (if available).

I don't think anyone follows this, AFAIK every engine puts non-persistent notification to the notification center, including Firefox, Chrome, Safari on Windows and macOS.

![Firefox and Chrome puts non-persistent notification to the Windows notification center](https://github.com/whatwg/notifications/assets/3396686/890ddc26-33aa-4154-aa76-ea057e3a2c65)

I see no reason to not put it there as long as the triggering page is still open. It's added by 0005987711586c6bf2ed20a7e3f157db18a9e9b3, so perhaps @annevk may remember the background behind this.

**Comment by annevk** [comment]
Yeah, seems reasonable to remove this recommendation. Should probably not discuss the specific experience too much as what makes sense can easily change over time.
